packages/docs/stories/templates/form.stories.tsx

```diff
@@ -0,0 +1,206 @@
+import { getLabel, getTranslation } from '../../src/translations.js';
+
+const meta: Meta = {
+  parameters: {
+    docs: {
+      page: () => (
+        <>
+          <Title />
+          <Subtitle />
+          <Description />
+          <Stories title="" />
+        </>
+      ),
+      story: {
+        iframeHeight: 1600,
+        inline: false,
+      },
+    },
+  },
+  title: 'Templates/Forms',
+};
+export default meta;
+
+export const ContactForm = {
+  parameters: {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    chromatic: {
+      ...storyBookPreviewConfig?.parameters?.chromatic,
+      disableSnapshot: false,
+    },
+  },
+  render: () => html`
+    <style>
+    .synergy-form-demo {
+      background: var(--syn-color-neutral-0);
+      margin: 0 auto;
+      padding: var(--syn-spacing-x-large);
+      max-width: 750px;
+    }
+
+    h1 {
+      font-size: var(--syn-font-size-3x-large);
+      font-weight: normal;
+      margin: 0 0 var(--syn-spacing-medium) 0;
+    }
+
+    .synergy-form-demo > p {
+      margin: 0 0 var(--syn-spacing-2x-large) 0;
+    }
+
+    em {
+      font-size: var(--syn-font-size-x-small);
+      font-style: normal;
+    }
+
+    fieldset {
+      border: none;
+      margin: 0 0 var(--syn-spacing-2x-large) 0;
+      padding: 0;
+    }
+
+    fieldset legend {
+      font-size: var(--syn-font-size-large);
+      margin-bottom: var(--syn-spacing-medium);
+    }
+
+    fieldset legend + p {
+      font-size: var(--syn-font-size-medium);
+      margin: 0 0 var(--syn-font-size-large) 0;
+    }
+
+    .fields {
+      container-type: inline-size;
+      display: flex;
+      flex-flow: wrap;
+      margin-bottom: var(--syn-spacing-2x-large);
+      gap: var(--syn-spacing-large) var(--syn-spacing-large);
+    }
+
+    .fields > * {
+      flex-basis: 100%;
+    }
+
+    @container (min-width: 640px) {
+      .fields > * {
+        flex-basis: calc(50% - var(--syn-spacing-medium));
+      }
+    }
+
+    /*
+     * The form control inputs are using a css grid to be displayed.
+     * We just let them flow automatically mobile and arrange them
+     * in two columns when space is wide enough
+     */
+    syn-radio-group {
+      container-type: inline-size;
+    }
+    syn-radio-group::part(form-control-input) {
+      display: grid;
+    }
+
+    @container (min-width: 640px) {
+      syn-radio-group::part(form-control-input) {
+        grid-auto-flow: column;
+        grid-template-rows: 1fr 1fr 1fr 1fr;
+      }
+    }
+
+    .submit-actions {
+      display: flex;
+      justify-content: right;
+      margin-bottom: var(--syn-spacing-2x-large);
+    }
+    </style>
+    <div class="synergy-form-demo">
+      <h1>${getTranslation('contactForm.headline')}</h1>
+      <p>${getTranslation('contactForm.subHeadline')}</p>
+
+      <form method="post" id="syn-form-demo">
+        
+        <fieldset>
+          <syn-radio-group
+            label="${getLabel('contactForm.topicLabel')}"
```

Comment:
Is there a specific reason why sometimes the `getTranslation` method and sometimes the `getLabel` method is used? 
I think it would be easier for others, if we keep the code as simple as possible and therefore only use one of them for translations.

Reply:
Reason for this was Typescript. When using `getTranslation`, you may either receive a string or an array of strings (for options). In this case, TS will yell at you as lits properties only accept strings. In this case you would always have to cast, e.g.

`label="${getLabel('contactForm.topicLabel')}"` will become:
`label="${(getLabel('contactForm.topicLabel') as string)}"` in all lines, makeing it kind of repetitive. This was the sole reason for this function.

Reply:
Update: Just had another look at it. Seems I was smart enough to circumvent this by updating `getTranslations`, but I did not remove `getLabel`. Will do that now :)